src/main/java/com/karlofduty/EMCRecipies/EMCRecipies.java

```diff
@@ -197,5 +197,15 @@ public void onEnable()
 
         this.getServer().addRecipe(new FurnaceRecipe(new ItemStack(Material.SMOOTH_SANDSTONE, 1), Material.SANDSTONE));
         this.getServer().addRecipe(new FurnaceRecipe(new ItemStack(Material.SMOOTH_RED_SANDSTONE, 1), Material.RED_SANDSTONE));
+        
+        /*
+            POPPED CHORUS FRUIT
+         */
+
+        final ShapedRecipe poppedchorusfruit = new ShapedRecipe(new NamespacedKey(this, "emc_poppedchorusfruit"), new ItemStack(Material.POPPED_CHORUS_FRUIT, 4));
```

Comment:
Should this really be a shaped recipe?

Reply:
No it should not have been, that was my mistake, an oversight.